schema.graphql

```diff
@@ -3,3 +3,23 @@ type PureProxy @entity {
   blockNumber: Int!
   extrinsicIndex: Int!
 }
+
+# An account can be a multisig, or a normal account (like a signatory)
+type Account @entity {
+  id: ID!
+  address: String!
+  # if this account is the signatory on one or many multisigs, this will link to an array of multisigs
+  multisigs: [AccountMultisig] @derivedFrom(field: "signatory")
+  isMultisig: Boolean
```

Comment:
whether it should be a mandatory field?

Reply:
Do you mean it should be mandatory? I think we will only request in the front-end `isMultisig: true`, and ignore the rest.
All in all, I don't really mind, but I don't think it absolutely has to.